src/style.css

```diff
@@ -0,0 +1,81 @@
+* {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  height: 300px;
+  min-width: 300px;
+  flex-shrink: 0;
+  flex-grow: 1;
+  flex-basis: calc(100% / 3);
+  font-size: 100px;
+  color: #fff;
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+}
+
+.block-1 {
+  background-color: rgb(100%, 0%, 0%);
+}
+
+.block-2 {
+  background-color: rgb(64%, 0%, 0%);
+  order: 1;
+}
+
+.block-3 {
+  background-color: rgb(48%, 0%, 0%);
+  order: 2;
+}
+
+.block-4 {
+  background-color: rgb(32%, 0%, 0%);
+  order: 3;
+}
+
+.block-5 {
+  background-color: rgb(16%, 0%, 0%);
+  order: 4;
+}
+
+.block-6 {
+  background-color: rgb(0%, 0%, 0%);
+  order: 5;
+}
```

Comment:
don't add these custom orders here, just change them in media rules, that'll simplify your code

Reply:
OK. Done it.